server/lib/util.py

```diff
@@ -345,6 +345,22 @@ def get_nl_chart_titles():
   return chart_titles
 
 
+class NlPropTitle:
+  # Display name for the prop
+  displayName: str
+  # String format to use when using the prop in a title
+  titleFormat: str
+
+
+# Returns display titles for properties used in NL
+# TODO: need to validate that every titleFormat has entity in it
+def get_nl_prop_titles() -> Dict[str, NlPropTitle]:
+  filepath = os.path.join(get_repo_root(), "config", "nl_page",
+                          "prop_titles.json")
+  with open(filepath, 'r') as f:
+    return json.load(f)
```

Comment:
I was just suggesting to document `dict`, but since you've defined the class, then how about making it `@dataclass` and doing:

```
  json_dict = json.load(f)
  return {k: NlPropTitle(**v) for k, v in json_dict.items()}
```

To make it the real returned type that you can access in triple.py?

Reply:
ohh, I'll just use a `dict` then, especially because format of the object may change as we decide how we want to do titles, updated